manifests/init.pp

```diff
@@ -1,259 +1,237 @@
-# == Class: corosync
+# @summary Configures the Pacemaker+Corosync stack to provide high-availability.
 #
 # This class will set up corosync for use by the Puppet Enterprise console to
 # facilitate an active/standby configuration for high availability.  It is
 # assumed that this module has been initially ran on a Puppet master with the
 # capabilities of signing certificates to do the initial key generation.
 #
-# === Parameters
-#
-# [*enable_secauth*]
+# @param enable_secauth [Boolean]
```

Comment:
Declaring the type here is redundant. See 
https://puppet.com/docs/puppet/5.5/puppet_strings_style.html#parameters

Reply:
Interesting. I must have been using an older build of puppet strings when I first started adding these tags. At the time it would print a warning for every parameter which was missing the type in the string declaration.